test/test_frame_2d.py

```diff
@@ -2,98 +2,89 @@
 import pytest
 
 
-@pytest.mark.physical_camera
-def test_image_context_manager(physical_camera_frame_2d):
```

Comment:
I think the idea is that when testing a module you import that module as part of the test itself. Then, if the import is broken it gets counted as a failure on all the tests that depend on that module.

I don't feel very strongly about this principle. It's just been in this repo since before I started here.
We could change it, but not in this PR :)

Reply:
That's not how imports work, right? It would be imported once the first time a function imports it, and the subsequent imports would have no effects. Seems a little strange we have a useless import per function there. 

I suppose it need not be addressed in this PR, though.